codice/ddf
# DDF-4588 Create a framework for standalone queries

#### What does this PR do?
- Feature flagging for standalone queries
- Create new route for standalone queries
- Basic framework for query operations

#### Who is reviewing it? 

@glenhein 
@cantstoptheunk 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@tbatie

#### How should this be tested?

- Ingest some query metacards
- Turn on standalone queries feature
- Verify that all queries show up on page
- Resize the page and verify that nothing unwanted happens

#### Any background context you want to provide?
- Queries are going to become standalone. This page provides the basic framework for such queries. 

#### What are the relevant tickets?
For GH Issues:
Fixes: #4588 

#### Screenshots

<img width="1920" alt="Screen Shot 2019-04-01 at 2 50 20 PM" src="https://user-images.githubusercontent.com/7646577/55359153-64846c00-548e-11e9-872d-b0354ce59ccf.png">
<img width="561" alt="Screen Shot 2019-04-01 at 2 50 31 PM" src="https://user-images.githubusercontent.com/7646577/55359159-664e2f80-548e-11e9-92b7-3af4b2beec1b.png">


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
